client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/AppiumCapabilitiesBuilder.java

```diff
@@ -44,14 +45,19 @@ public DesiredCapabilities getCapabilities(DesiredCapabilities capabilities) {
         }
         capabilities.setCapability(MOBILE_NODE_TYPE, mobileSession.getMobileNodeType().getAsString());
         capabilities.setCapability(MobileCapabilityType.DEVICE_NAME, mobileSession.getDeviceType());
+        if (mobileSession.getDeviceType() != null) {
+            capabilities.setBrowserName(mobileSession.getDeviceType().toLowerCase());
+        }
         capabilities.setCapability(MobileCapabilityType.PLATFORM_VERSION, mobileSession.getPlatformVersion());
         capabilities.setCapability(APPIUM_LOCALE_TYPE, mobileSession.getAppLocale());
         capabilities.setCapability(APPIUM_LANGUAGE_TYPE, mobileSession.getAppLanguage());
         if (mobileSession.getPlatform() == WebDriverPlatform.ANDROID) {
             capabilities.setCapability(MobileCapabilityType.PLATFORM_NAME, APPIUM_ANDROID_PLATFORM_TYPE);
+            capabilities.setPlatform(Platform.ANDROID);
```

Comment:
for sure that will solve the problem. i am gonna remove platform capability too.